src/app/beer_garden/api/http/handlers/v1/system.py

```diff
@@ -312,7 +312,8 @@ async def get(self):
                     "exclude_fields": exclude_fields,
                     "dereference_nested": dereference_nested,
                 },
-            )
+            ),
+            serialize_kwargs=serialize_kwargs,
```

Comment:
If you're adding this here, shouldn't you remove the `serialize_kwargs` from the operation kwargs?

Reply:
You are right, that is redudent